### ISSUE TITLE
How many samples should be used in the analysis

### ISSUE THREAD

**Issue by enluo211** [stschiff/msmc2#61]
When I used 16 haplotypes for analysis(two poputation,four samples per population), the number of bits in my MSMC input file was very small, only a few hundred, when I used 8 haplotypes for analysis, the number of bits in the MSMC input file was several thousand (about 10 times the number of 16 haplotypes).Will too few sites in the input file affect the result? If so, how can I avoid this from happening?
[chr1.8hapmap.multihetsep.txt](https://github.com/stschiff/msmc2/files/15056380/chr1.8hapmap.multihetsep.txt)
[chr1.16hapmap.multihetsep.txt](https://github.com/stschiff/msmc2/files/15056381/chr1.16hapmap.multihetsep.txt)

**Comment by stschiff** [comment]
Well, your files have extremely different lengths. One has a few dozen sites, the other many more. You seem to be missing a hole bunch of sites from your 16-haplotype input file.

**Comment by enluo211** [comment]
Yeah,but I don't know what causes this phenomenon

**Comment by enluo211** [comment]
Is it because most loci are masked, and if so, why does this happen?

**Comment by stschiff** [comment]
Well, but how can I possibly know what went wrong with the creation of your file?

**Comment by enluo211** [comment]
You are right, but all my operations follow the process provided by you on github (refer to msmc2 and msmc-tools).

**Comment by enluo211** [comment]
[MSMC-parv-cluster20240421.txt](https://github.com/stschiff/msmc2/files/15079487/MSMC-parv-cluster20240421.txt)
the code for this step is just like this

**Comment by enluo211** [comment]
[beagle-phase.txt](https://github.com/stschiff/msmc2/files/15079628/beagle-phase.txt)
[genome-mask.txt](https://github.com/stschiff/msmc2/files/15079629/genome-mask.txt)
[vcf-mask.maize.txt](https://github.com/stschiff/msmc2/files/15079630/vcf-mask.maize.txt)
[makeMappabilityMask-Zm-py.txt](https://github.com/stschiff/msmc2/files/15079679/makeMappabilityMask-Zm-py.txt)
I used these scripts to generate the vcf-mask, genome-mask, and vcf-phase files, respectively
